dashboard/test/helpers/delete_accounts_helper_test.rb

```diff
@@ -256,6 +256,29 @@ class DeleteAccountsHelperTest < ActionView::TestCase
     assert user.valid?
   end
 
+  #
+  # Table: dashboard.user_permissions
+  #
+
+  test "revokes the user's permissions" do
+    user = create :teacher
+    UserPermission::VALID_PERMISSIONS.each {|perm| user.permission = perm}
+    refute_empty UserPermission.where(user_id: user.id)
+
+    purge_user user
+
+    assert_empty UserPermission.where(user_id: user.id)
```

Comment:
Maybe worth noting that this actually happens on soft-delete, and this test passes with or without the "revoke all permissions" call in the purge_user operation.